UPGRADE_NOTES.md

```diff
@@ -1,3 +1,90 @@
+## Upgrading from 0.27 to 0.28
+
+For a full diff of necessary changes, please see [LuckyDiff](https://luckydiff.com?from=0.27.2&to=0.28.0).
+
+- Upgrade Lucky CLI (homebrew)
+
+```
+brew update
+brew upgrade lucky
+```
+
+- Upgrade Lucky CLI (Linux)
+
+> Remove the existing Lucky binary and follow the Linux
+> instructions in this section
+> https://luckyframework.org/guides/getting-started/installing#on-linux
+
+- Update versions in `shard.yml`
+  - Crystal should be `">= 1.0.0"`
+  - Lucky should be `~> 0.28.0`
+  - Authentic should be `~> 0.8.0`
+  - Carbon should be `~> 0.2.0`
```

Comment:
`carbon_sendgrid_adapter ~> 0.1.0` should be included if you're using sendgrid to send emails